ocs_ci/helpers/dr_helpers.py

```diff
@@ -532,21 +584,22 @@ def wait_for_replication_resources_deletion(namespace, timeout, check_state=True
             logger.info(error_msg)
             raise TimeoutExpiredError(error_msg)
 
-    logger.info("Waiting for VRG to be deleted")
-    sample = TimeoutSampler(
-        timeout=timeout, sleep=5, func=check_vrg_existence, namespace=namespace
-    )
-    if not sample.wait_for_func_status(result=False):
-        error_msg = "VRG resource not deleted"
-        logger.info(error_msg)
-        raise TimeoutExpiredError(error_msg)
+    if "cephfs" not in namespace:
+        logger.info("Waiting for VRG to be deleted")
```

Comment:
In which line did we trigger the VRG deletion?

Reply:
This is not a new code. I think it will be automatic for RBD. This PR just skips this check for CephfS.

Reply:
@sidhant-agrawal Any idea?